packages/documentsearch/src/searchview.tsx

```diff
@@ -265,19 +266,41 @@ function ReplaceEntry(props: IReplaceEntryProps): JSX.Element {
 }
 
 interface IUpDownProps {
+  commands: CommandRegistry;
   onHighlightPrevious: () => void;
   onHighlightNext: () => void;
   trans: TranslationBundle;
 }
 
 function UpDownButtons(props: IUpDownProps) {
+  // Getting the key bindings for the next and previous commands
+  const nextBinding = props.commands.keyBindings.find(
+    (binding: { command: string }) =>
+      binding.command === 'documentsearch:highlightNext'
+  );
+
+  const prevBinding = props.commands.keyBindings.find(
+    (binding: { command: string }) =>
+      binding.command === 'documentsearch:highlightPrevious'
+  );
+
+  const nextKeys = nextBinding
+    ? CommandRegistry.formatKeystroke(nextBinding.keys)
+    : '';
+  const prevKeys = prevBinding
+    ? CommandRegistry.formatKeystroke(prevBinding.keys)
+    : '';
+
+  const prevShortcut = prevKeys ? `(${prevKeys})` : '';
+  const nextShortcut = nextKeys ? `(${nextKeys})` : '';
```

Comment:
Let's add the space here to avoid adding one if no shortcut exist

```suggestion
  const prevShortcut = prevKeys ? ` (${prevKeys})` : '';
  const nextShortcut = nextKeys ? ` (${nextKeys})` : '';
```